README.md

```diff
@@ -1,6 +1,5 @@
-[![Build Status](https://travis-ci.org/artur-tamazian/avro-schema-generator.svg?branch=master)](https://travis-ci.org/artur-tamazian/avro-schema-generator)
-[![Coverage](https://codecov.io/gh/artur-tamazian/avro-schema-generator/branch/master/graph/badge.svg)](https://codecov.io/gh/artur-tamazian/avro-schema-generator)
```

Comment:
I know it's not a big deal, but do you know any alternatives we could use?

Reply:
Sure, JaCoCo is free, up to date and widely used 
https://www.eclemma.org/jacoco/

Reply:
Would you mind migrating to it to keep the codecov.io coverage info?

Reply:
Unfortunately, i dont know maven well for such sort of things and would like to concentrate on fixing #7 instead.
The main reason for getting tid of cubertura is that cubertura task hides test failures, so even if tests are failing of missing dependencies - the task and the build are passing succesfully.
This bug is more critical than code coverage IMO.

Reply:
Got it. Hopefully, I'll find some time to look into it later.